staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -62,6 +62,42 @@ var (
 		},
 		[]string{"resource"},
 	)
+	watchersCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name: "apiserver_created_watchers",
+			Help: "Counter of watchers created in watchcache broken by resource type",
+		},
+		[]string{"resource"},
+	)
+	watchersGauge = metrics.NewGaugeVec(
```

Comment:
If I'm not missing anything, this is duplication of: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L155

If so, please remove.

Reply:
I'll remove this metric.